ci/compile.sh

```diff
@@ -46,6 +46,15 @@ export CGO_ENABLED=1
 GOARCH="${ARCHS_GO["${ARCH}"]}"
 export GOARCH="${GOARCH}"
 
+if [[ "${#}" -ge 1 ]]; then
+  if [[ "${1}" == "docker" ]]; then
+    # Inside container, go tried to put cache under HOME, and it's owned
```

Comment:
Is this still true? Aren't build containers using `root`?

Reply:
You're right, it's not needed anymore. Removed